.github/workflows/rawls-build-tag-publish-and-run-tests.yaml

```diff
@@ -0,0 +1,137 @@
+name: rawls-build-tag-publish-and-run-tests
+
+on:
+  pull_request:
+    paths-ignore: ['**.md']
+  push:
+    branches:
+      - develop
+    paths-ignore: ['**.md']
+
+jobs:
+  rawls-build-tag-publish-job:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: 'read'
```

Comment:
What are these for?

Reply:
From time to time, the jobs or steps within a GitHub Action workflow may need permissions to read or write artifacts (e.g. access tokens) into local directories of the workflow runner machine. This gives the necessary permissions.